### PR TITLE
data: mikejw

### PR DESCRIPTION
**Pull request by mikejw**

### CODE REVIEW

**Review by himanshu1221 — CHANGES_REQUESTED** [review]
hey @mikejw `avatar` has been depreciated please remove it

**Comment by mikejw** [comment]
Hi @himanshu1221 that's done. Cheers.

File: data/mikejw.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "links": [
+    {
+      "name": "Portfolio",
+      "url": "https://www.mikejw.co.uk",
+      "icon": "FaLink"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/mikeyjw",
+      "icon": "FaLinkedIn"
```
Comment (himanshu1221) [inline data/mikejw.json:25]: Hey @mikejw please update this `FalinkedIn` to `FaLinkedin`

**Review by himanshu1221 — CHANGES_REQUESTED** [review]

File: data/mikejw.json (hunk 1 above)
Comment (mikejw) [reply data/mikejw.json]: thanks. done!

**Review by himanshu1221 — APPROVED** [review]
Awesome :rocket:  LGTM :+1:

**Review by khushalarora — APPROVED** [review]
Looks good, thanks  :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**